submissions/DmytryjK/task_js_dom/js/script.js

```diff
@@ -0,0 +1,169 @@
+        imgAlt : "Grenland canyon",
+        firstParagraph : "Цей воістину дивовижний острів притягує мандрівників з усього світу, як магніт.  Аналогія банальна, але краще вигадати складно. Значна частина Гренландії справді схована під надійним щитом із льоду, який останнім часом починає активно танути. Це результат глобального потепління, на яке вже багато років намагаються звернути увагу багато громадських організацій.",
+        secondParagraph : "У льодовиках утворюються численні каньйони - їх краса просто приголомшує. Таке враження, що природа, як найталановитіший художник на цій планеті, малює своїм пензлем хитромудрі, часом навіть сюрреалістичні картини."
+    },
+    {
+        h2 : "Норвегія",
+        subtitle : "Місто - Олесунн",
+        imgSrc : "images/Alesund norway.jpg",
+        imgAlt : "Alesund norway",
+        firstParagraph : "О́лесунн (норв. Ålesund) — місто і порт на заході Норвегії. Розташоване на північ від гирла фіорду Стур. Місто розкинулося на декількох островах, а саме — Норвьойя, Аспьойя, Гейсса (Гесса) та Окснойя, які є з'єднані мостами. За легендою, поселення бере свій початок у 9 столітті коли Ролло (Рольф) заснував поблизу маєток, де мешкав війт. Статус міста Олесунн отримав в 1848 р. Після того як місто зазнало пожежі в 1904 р., воно було відбудоване у оригінальному стилі арт-нуво.",
+        secondParagraph : "Олесунн — регіональний торговий центр і туристична база для маршрутів в регіонах Суннмьоре, долині Норанґ, льодовиках Ойє та островах Рунде та Ґіске. В місті знаходиться найбільша риболовна гавань Норвегії, пристановище для риболовних трейлерів що виловлюють тріску та палтус з виду камбалових. Разом з Тромсьо, Олесунн є осередком ловлі арктичних тюленів."
+    }
+];
+
+document.addEventListener('DOMContentLoaded', startEventsForPage());
+function startEventsForPage() {
+    //Tabs
+    const mainContentBlock = document.querySelectorAll('.main__content-block');
+    const navigationButton = document.querySelectorAll('.main__navigation-button');
+
+    createElements();
+
+    addClass(mainContentBlock, 'main__content-active');
+
+    currentTabButton('main__navigation-button_active');
+
+    function currentTabButton (tabSelector) {
+        addClass(navigationButton, tabSelector);
+        navigationButton.forEach((button, index) => {
+            button.addEventListener('click', e => {
+                e.preventDefault();
+                removeClass(navigationButton, tabSelector);
+                e.currentTarget.classList.add(tabSelector);
+
+                removeClass(mainContentBlock, 'main__content-active');
+
+                deleteInnerHtml();
+                createElements(index);
+
+                addClass(mainContentBlock, 'main__content-active', index);
+            });
+        });
+    };
+
+    function createElements (index = 0) {
+        mainContentBlock[index].innerHTML = `
+            <h2 class="main__content-title">${contentArray[index].h2}</h2>
```

Comment:
You must update only one element.

Reply:
Yep, but i have some empty blocks thats why i have to use index. The reason, why i do this - couse i whant add animation class, when my content show on the website. 
At first I created 1 block and generated content in it, but then I could not manipulate the animation. Perhaps you can advise how I can add animation to an element after it has been rendered in the container? Google did not help to find the answer ((